db/migrations/1138.add_mivideo_tables.py

```diff
@@ -0,0 +1,14 @@
+from yoyo import step
```

Comment:
The migration number in the file name will need to be updated. Also, in recent migrations, we've been using templated comments at the top and specifying the previous migration as a dependency. Let me know if you have questions/concerns.

Reply:
Questions:

1. I wasn't sure how many migrations there would be between the time I started working on the MiVideo implementation and the time my PR would get merged.  So I chose a sufficiently high number (`1138`).  Should I switch it to `0012` now?  Is there a convention I can use to reserve that number for the migration in my PR?
2. What is the significance of comments like `# file: migrations/0011.add_course_sis_ids.py`?  Is it necessary?  Isn't the file's own name sufficient to express this information?
3. I didn't think my implementation's migration depended on any earlier migrations, but thinking about it now, I guess it depends on `0009.add_meta_tables.py` for `job_run`.  Does that make sense?
4. Should I try to add some relationships between `mivideo_media_started_hourly` and other tables, like `course`?  That would also make dependencies between the jobs.  I'd no longer be able to run the MiVideo job without running the job that populates `course`, too.

Reply:
Will get these answered after my meeting.

Reply:
I would not add any relationships at this point, since this is an append table and course is not.

Reply:
I've changed…

* …the name to `0012…`.
* …the dependency to use `0009…`.

Reply:
1) There is no convention for reserving migration numbers yet, nor do I think it's really that necessary to have one. It's fairly straightforward to resolve any conflicts, especially since you will have named your migration differently (meaning the merge won't obliterate your file) and then you will just need to bump the number.

2) There isn't really any significance to the comment, other than extra clarity. I think the format came from the `yoyo` documentation. I didn't start off using it, I think @pushyamig  did first.

3) I think we've been fairly lackadaisical about the use of `__depends__`.  Recently to keep things simple we've just been specifying the most recent migration as the only dependency, regardless of whether it touches any of the same tables. From a technical database-oriented perspective, I don't think your migration has any dependencies. If you want to be consistent, I'd just use the `0011` migration. I don't think the metadata table migration is a dependency of this one.

Reply:
✅Fixed.

Not adding the comment with embedded file name.  It's superfluous.  Also, if the name of the file or the directory changes later, will someone seriously go and update all the comments?

Reply:
That's fine, I'm not insisting on. FWIW, the path in the comment shouldn't change, since `yoyo` basically requires that these files be stored in a directory called `migrations`.